code/game/objects/structures/kitchen_spike.dm

```diff
@@ -42,17 +40,21 @@
 	meat = 5
 	return 1
 
-/obj/structure/kitchenspike/attack_hand(mob/user as mob)
+/obj/structure/kitchenspike/attack_hand(mob/living/carbon/user)
 	if(..() || !occupied)
 		return
 	meat--
 	new meat_type(get_turf(src))
-	if(src.meat > 1)
+	if(meat > 1)
 		to_chat(user, "You remove some meat from \the [victim_name].")
-	else if(src.meat == 1)
+	else if(meat == 1)
 		to_chat(user, "You remove the last piece of meat from \the [victim_name]!")
 		icon_state = "spike"
 		occupied = 0
+	if(meat_type == user.species.meat_type)
+		user.sanity_damage += 5*((user.nutrition ? user.nutrition : 1)/user.max_nutrition) // The more hungry the less sanity damage.
+		to_chat(user, SPAN_NOTICE("You feel your [user.species.name]ity dismantling as you slabs off \the [src]")) // Human-ity , Monkey-ity , Slime-Ity
```

Comment:
shouldn't it be slab rather than slabs ?

Reply:
fixed , thanks for pointing out.